scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -413,7 +479,8 @@ private void initVarsLayout() {
         varsLayout = new VLayout();
         varsLayout.setIsGroup(true);
         varsLayout.setGroupTitle("2. Fill workflow variables");
-        varsLayout.setWidth100();
```

Comment:
Why can't we let `setWidth` and just add `setHeight` (insteand of replacing `setWidth` by `setHeight`)?

Reply:
I make it.
I add setHeight

Reply:
Ok, I just wanted you to notice that you removed the width resizing that matches the size of the window. If it still displays correctly then it is fine.